xcube_cmems/version.py

```diff
@@ -19,4 +19,4 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
I'd rather stay with the numbering, as _not_ seem to have changes the library's API but instead fixed it towards compatibility with an external API:

```suggestion
version = "0.1.7"
```